src/types/annotator.ts

```diff
@@ -49,6 +49,19 @@ export type DocumentMetadata = {
   documentFingerprint?: string;
 };
 
+/**
+ * Identifies a loadable chunk or segment of a document.
+ *
+ * Some document viewers do not load the whole document at once. For example
+ * an EPUB reader will load one Content Document from the publication at a time.
+ */
+export type SegmentInfo = {
+  /** Canonical Fragment Identifier for an EPUB Content Document */
+  cfi?: string;
+  /** Relative or absolute URL of the segment. */
+  url?: string;
+};
```

Comment:
It might be useful to add other fields here in future, such as the chapter title if available. For now only the information that is most useful for matching annotations against the current segment is included.